### DISCUSSION TITLE
Invalid copilot token

### DISCUSSION THREAD

**Discussion by YafeshanUnal** [community/community#72714] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello everyone, I'm getting this error in visual studio code, I tried input and output and a lot of things but it didn't fix it, can you help me?
`Invalid copilot token: missing token: 403 `

**Comment by YafeshanUnal** [comment, marked as answer]
I solved this problem by opening a ticket to google support

**Comment by ghostinhershell** [comment]
Hi @YafeshanUnal,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Just a heads up that your duplicate post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

Thanks!
